homework.js

```diff
@@ -0,0 +1,88 @@
+// Q1:
+
+function sumNum(n) {
+    let total = 0;
+      for(let i = 1; i <= n; i++){
+        total += i;
+      }
+      return total;
+  }
+
+//Q2:
+
+function sumEven(n) {
+    let total = 0;
+      for(let i = 1; i <= n; i++){
+        if(i % 2 === 0) {
+            total= total+ i;
+            }
+            }
+            return total;
+            }
+
+//Q3:
+
+function avr(num) {
+    var total = 0, i;
+    for (i = 0; i < num.length; i += 1) {
+        total += num[i];
+    }
+    return total / num.length;
+}
+
+//Q4:
+
+function reverseWord(str){
+    let rev ="";
+    for(let i= 0; i<str.length; i++){
+        rev = str[i]+rev;
+    }
+    return rev;
+}
+
+//Q5:
+
+let b = ["Hi","World"];
+
+console.log(b.join('-'));
+
+//Q6:
+
+
+function count(n) {
+    let arr = [];
+    for (let i = 1; i<=n; i++) {
+        arr.push(i.toString());
+
+       
+    }
+
+    let j = n-1;
+    for (j ; j>=i; i-- )
```

Comment:
since you want to print the numbers from `j` to `1`, your stop condition should be `j>0` 
 and since each time you going to push `j` you should decrease `j` each step.
 so your for loop should looks like this in the end `for (j ; j>0; j-- ) `